bucket/rw/json.py

```diff
@@ -0,0 +1,86 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2023-2024 Vypercore. All Rights Reserved
+
+import json
+from pathlib import Path
+
+from .common import (
+    AxisTuple,
+    AxisValueTuple,
+    BucketGoalTuple,
+    BucketHitTuple,
+    GoalTuple,
+    PointHitTuple,
+    PointTuple,
+    Reading,
+    Writer,
+)
+
+###############################################################################
+# Accessors
+###############################################################################
+
+
+class JSONWriter(Writer):
```

Comment:
Note only reason we added a JSON writer is because JSON is very easy to consume for web stuff